tests/contrib/grpc/test_grpc.py

```diff
@@ -572,3 +574,40 @@ class NotFutureLike(object):
     assert span.duration is None
     _handle_response(span, FutureLike())
     assert span.duration is not None
+
+
+@pytest.fixture()
+def patch_grpc():
+    patch()
+    try:
+        yield
+    finally:
+        unpatch()
+
+
+class _UnaryUnaryRpcHandler(grpc.GenericRpcHandler):
+    def __init__(self, handler):
+        self._handler = handler
+
+    def service(self, handler_call_details):
+        return grpc.unary_unary_rpc_method_handler(self._handler)
+
+
+@snapshot(ignores=["meta.grpc.port"])
```

Comment:
This snapshot could also catch the server spans if we wanted with an `async_mode=False`.

@majorgreys should we also capture the server span?

Reply:
The server and client, in this run at least, share the same global tracer (through the pin) running in the same process. WDYT?

Reply:
Ohh the server span is there! I missed it because of the matching operation name